2021/34xxx/CVE-2021-34853.json

```diff
@@ -1,18 +1,70 @@
 {
```

Comment:
'https://www.foxit.com/support/security-bulletins.html\xa0' does not match '^(ftp|http)s?://\\S+$'